specification/authorization/resource-manager/readme.md

```diff
@@ -259,6 +259,15 @@ input-file:
 - Microsoft.Authorization/stable/2015-07-01/authorization-RoleAssignmentsCalls.json
 ```
 
+### Tag: package-2021-01-01-preview
```

Comment:
should the tag be package-2021-01-01-preview-only if only one swagger? could user complete an end to end scenario with only this one swagger file?

Reply:
Yes, for this specific API, the users are approvers of roleAssignmentApprovals. They can complete all actions necessary using the roleAssignmentApproval APIs. They don't have to deal with roleAssignment or any other entity

Reply:
Thanks for bringing this up! I've renamed to 2021-01-01-preview-only